README.md

```diff
@@ -149,18 +147,18 @@ Besides being a valuable item of information in and of itself, it allowed me to
 
 *Table 5 Detailed distribution of entries per certainty score range in the 'High-confidence' subset*
 
-Next (this further narrowing down of the subsets was mainly due to the high number of entires especially in the second subset), I took a random sample of approximately 270 entries (guided by: 10 % of 'Low-confidence' subset). The distribution across the 'Score ranges' (See Table 4 above) was weighted by the respective number of entries in these; i.e., for the 'Low-confidence' subset:
+Next (this further narrowing down of the subsets was mainly due to the high number of entries especially in the second subset), I took a random sample of approximately 270 entries (guided by: 10% of 'Low-confidence' subset). The distribution across the 'Score ranges' (See Table 4 above) was weighted by the respective number of entries in these; i.e., for the 'Low-confidence' subset:
```

Comment:
I think it would be easier to read if the parenthetical clause was its own sentence